nodejs/webapi/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "webapi",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
Would you be able to add `build` and `start` script commands here, like:

- `"build": "npm build"`
- `"start": "node server.js"`